### ISSUE TITLE
1.11b2: Status window is also displayed when not configured

### ISSUE THREAD

**Issue by SanderGit** [rcaelers/workrave#444]
**Describe the bug**
On 1.11b2, the status window is displayed, also when it has not been configured to do so (i.e. only the taskbar applet should be visible)

** Windows (please complete the following information in case you encountered the bug on Windows):
 - Windows Version: Windows 10
 - Workrave Version 1.11b2

**Comment by rcaelers** [comment]
Thanks for testing the 1.11 beta!

**Comment by SanderGit** [comment]
Issue still occurs in beta 5 @rcaelers

**Comment by rcaelers** [comment]
Indeed. I think it is fixed now.

**Comment by gringus** [comment]
@rcaelers I tested latest `master` on Gentoo Linux (`XFCE 4.19` environment) a few days ago and noticed the following behavior: when I start _Workrave_ interactively in the terminal _Workrave_ window is not shown (only systray icon), but when it is started during login process (autostart) _Workrave_ window still pops up. This was not observed with _Workrave_ `1.10.50`. Any idea why this is?

**Comment by gringus** [comment]
Current workaround:
`touch ~/.workrave/workrave.ini`
This way configuration is stored/read from custom configuration file.

**Comment by SanderGit** [comment]
This should be fixed on 1.11b6. But here on Windows, the preference to show the status window seems to re-enable itself sometimes. Can anyone confirm?

**Comment by Chealer** [comment]
@SanderGit: Are you still experiencing issues with 1.11.1 or above?

**Comment by SanderGit** [comment]
> [@SanderGit](https://github.com/SanderGit): Are you still experiencing issues with 1.11.1 or above?

No. On Windows 11 the taskbar applet is not supported by default anyway.
